### PR TITLE
Update segment_web_page_views__sessionized.sql

### PR DESCRIPTION
**Pull request by Patchus**
new db-utils is actually moving to list views, so this will break (kind of already is). This will just prep it for the eventual depreciation.

### CODE REVIEW

**Comment by Patchus** [comment]
The first commit deals with the new Utils only allowing [] not strings.
The second commit deals with a distinct on the referral table. If there are duplicates it will fan out the actual results and cause duplicates that will prevent the merge/update
The third commit actually removes the duplicate; but the second commit should prevent the third from becoming an issue

File: data/referrer_mapping.csv [hunk 1]
```diff
@@ -55,7 +55,6 @@ social,Hocam.com,hocam.com
 social,Hyves,hyves.nl
 social,Taringa!,taringa.net
 social,Classmates,classmates.com
-social,Pinterest,pinterest.com
```
Comment (clrcrl) [inline data/referrer_mapping.csv:58]: I'm not clear on why this line got removed

File: models/sessionization/segment_web_page_views__sessionized.sql [hunk 2]
```diff
@@ -130,7 +130,7 @@ session_ids as (
 
         {{dbt_utils.star(ref('segment_web_page_views'))}},
         page_view_number,
-        {{dbt_utils.surrogate_key('anonymous_id', 'session_number')}} as session_id
```
Comment (clrcrl) [inline models/sessionization/segment_web_page_views__sessionized.sql:133]: This would require a bump to the dbt-utils version number in the `packages.yml` file, otherwise it will break!

File: models/sessionization/segment_web_sessions__initial.sql [hunk 3]
```diff
@@ -120,6 +120,7 @@ tiers as (
 mapped as (
 
     select
+        distinct
```
Comment (clrcrl) [inline models/sessionization/segment_web_sessions__initial.sql:123]: Can you talk me through the `distinct` here — is this solving a separate issue (if so, it should be in a separate PR!). `distinct`s are expensive, so I'd prefer to avoid them if possible :)

**Review by clrcrl — CHANGES_REQUESTED** [review]
Hi @Patchus — thanks for your contribution. A few clarifying questions though!

File: data/referrer_mapping.csv (hunk 1 above)
Comment (Patchus) [reply data/referrer_mapping.csv:58]: No worries! Line 58 is a duplicate of 1755 (when the pinterest category got expanded). If they are not unique, it will cause a duplicate join which prevents uniqueness in the warehouse. So I removed it, because this was added later in 1755.

File: models/sessionization/segment_web_page_views__sessionized.sql (hunk 2 above)
Comment (Patchus) [reply models/sessionization/segment_web_page_views__sessionized.sql:133]: Well the inverse is also true, if you bump dbt-utils, this breaks. I wonder if there is a compromise here? I upgraded to 15.2 and it caused this line of code to break.

File: models/sessionization/segment_web_sessions__initial.sql (hunk 3 above)
Comment (Patchus) [reply models/sessionization/segment_web_sessions__initial.sql:123]: Yeah the distinct here is to prevent the duplicate pintrest listed above. I think it is bold to assume that the mappings from referrer_mapping.csv will be unique. 

```select
        distinct
        tiers.*,
        referrer_mapping.medium as referrer_medium,
        referrer_mapping.source as referrer_source

    from tiers

    left join referrer_mapping on tiers.referrer_host = referrer_mapping.host
```

The join on host will duplicate the tiers and will cause non-unique referrals when they were unique prior to the join

File: models/sessionization/segment_web_page_views__sessionized.sql (hunk 2 above)
Comment (clrcrl) [reply models/sessionization/segment_web_page_views__sessionized.sql:133]: Oh I mean, we should do both. As in, if we use the `[]` syntax, let's make sure the `packages.yml` is updated to reflect that!

Comment (Patchus) [reply models/sessionization/segment_web_page_views__sessionized.sql:133]: ohh nice, yea that is a good idea!

**Comment by clrcrl** [comment]
Closing in favor of #53, which removes the duplicate and adds a test to ensure it doesn't sneak in
